README.md

```diff
@@ -1,5 +1,5 @@
 Joern - The Bug Hunter's Workbench
-===
+=== 
```

Comment:
What's this?

Reply:
random Js test case was failing, so this space was to retrigger checks

Reply:
ok, removing this